corehq/apps/app_manager/models.py

```diff
@@ -1347,6 +1347,11 @@ def get_submit_label(self, lang):
             return self.submit_label[lang]
         return 'Submit'
 
+    def get_submit_notification_label(self, lang):
+        if lang in self.submit_notification_label:
```

Comment:
Does this work if `self.submit_notification_label` doesn't exist yet? Or maybe it's already known to exist with its default value based on the model change -- not certain how that works.

Reply:
good call added check https://github.com/dimagi/commcare-hq/pull/33885#:~:text=in%20getter%20function-,9a288ed,-Merge%20state